packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -227,39 +210,60 @@ export default {
 }
 
 .kt-input-number {
+	display: flex;
+	align-items: center;
+
+	width: inherit;
```

Comment:
@carsoli I really think this should be `100%` instead. `inherit` just copies the parent value, which e.g. means that if the parent has `width: 50%`, the `KtInputNumber` would have another `width: 50%`, making it `25%` wide in total.

See here: https://stackoverflow.com/a/9374612/2857873